interface/super/edit_layout.php

```diff
@@ -573,16 +573,22 @@ function encodeModifier($jsonArray)
                 " form_id = '" . add_escape_custom($_POST['layout_id']) . "' " .
                 " AND field_id IN (";
     $comma = "";
+    $cntr = 0;
     foreach (explode(" ", $_POST['selectedfields']) as $onefield) {
-        $sqlstmt .= $comma . "'" . add_escape_custom($onefield) . "'";
-        $comma = ", ";
+        if (!isColumnReserved(tableNameFromLayout($_POST['layout_id']), $onefield)) {
+            $sqlstmt .= $comma . "'" . add_escape_custom($onefield) . "'";
+            $comma = ", ";
+            $cntr++;
+        }
```

Comment:
imo the wrong place to do this. should be in the validation of rename or add function.
It's already there in code it's just there are some sql errors that when fix will fix issue.

I'll show where once I fix errors. cool

Reply:
Sorry @stephenwaite you're correct I forgot the layouts table delete.
Still though there is another bug which is probably some of the recent layout editor issue.
L-287 all show table queries.

Reply:
ok, nice, saw that, this adodb execute seems familiar, like something happened with that library

Reply:
I think the bug is with non-alpha numeric named fields?